server/src/utils/BaseController.js

```diff
@@ -44,9 +44,7 @@ export class BaseController {
       skip,
       limit;
     
-    if (query) {
-      filter = {...filter, ...query}
-    }
+    filter = {...filter, ...query}
 
     if (req.query.filter) {
```

Comment:
query will still be an empty object here, which will coerce `req.query.filter` into this `if` block